### PR TITLE
fix CE processing starting before Views or Devcockpit at startup

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/computation/queue/CeQueueInitializer.java [hunk 1]
```diff
@@ -47,7 +49,14 @@ public CeQueueInitializer(DbClient dbClient, CEQueueStatus queueStatus, CeQueueC
   }
 
   @Override
-  public void start() {
+  public void onServerStart(Server server) {
+    if (!done) {
```
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/computation/queue/CeQueueInitializer.java]: For my information, why do you need to store this done state ? Can the onServerStart() be called many times on the same ServerStartHandler ?

Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/computation/queue/CeQueueInitializer.java]: Ok, I've got my answer below, with the UT you've added.

**Comment by julienlancelot** [comment]
LGTM

File: server/sonar-server/src/main/java/org/sonar/server/computation/queue/CeQueueInitializer.java (hunk 1 above)
Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/computation/queue/CeQueueInitializer.java]: this is code for the sole purpose of testing (in medium tests). I think it's crapy but I couldn't find an alternative

Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/computation/queue/CeQueueInitializer.java]: Indeed, we'll maybe need to update MT later.

**Pull request merged**